tests/jenkins/Jenkinsfile

```diff
@@ -2,6 +2,14 @@ pipeline {
     agent {
         label 'bb5'
     }
+    parameters {
+        string(name: 'sha1', defaultValue: 'master',
```

Comment:
Like we did in blueconfigs. We dont need to declare the variables that are passed. We only need to declare the variables that we want the user to edit. If we want to give the user the possibility of choosing a different commit  maybe we could use a more meaningful var name. In the logic we can then check both variables.

Reply:
True. I kept `sha1` because this is the variable to select the `CoreNeuron` branch but we can add another one (`CORENEURON_BRANCH`) to expose to the user and then select the branch using this one.

Reply:
> I kept sha1 because this is the variable to select the CoreNeuron branch

I meant we can remove from the `parameters`. It is anyway always passed as an environment variable

Reply:
True true. I will handle this parameter like we do in the `SimulationStack`.